metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -394,6 +398,38 @@ def get_dataset(
         # None/details
         return new_powerbi_dataset(workspace_id, response_dict)
 
+    def get_dataset_parameters(
+        self, workspace_id: str, dataset_id: str
+    ) -> Optional[Dict[str, str]]:
+        dataset_get_endpoint: str = RegularAPIResolver.API_ENDPOINTS[
```

Comment:
Should we wrap in try-except in case of failing to hit the endpoint?

Reply:
Don't want failure here to interrupt other things

Reply:
Nvm see it's guarded below!

Reply:
it's wrapped in a try-except at the call site